pkgs/tools/misc/ets/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "ets";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "zmwangx";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-XWIDo5msTMTLr60x1R9cwsiZIDG6G+uHWx8idt4F2iA=";
+  };
+
+  patches = [ ./go-mod.patch ];
```

With a PR in place this makes it easier to figure out when to drop the patch:

```suggestion
  # https://github.com/zmwangx/ets/pull/18/
  patches = [ (fetchpatch { 
    url = "https://github.com/zmwangx/ets/commit/600ec17a9c86ca63cd022d00439cdc4978e2afa9.patch";
    sha256 = "<TODO>";
  }) ];
```

---

Thanks for pointing out fetchpatch